src/components/Navbar.js

```diff
@@ -49,17 +49,17 @@ export default function Navbar() {
                 <div className="hidden sm:ml-6 sm:block">
                   <div className="flex space-x-4">
                     {navigation.map((item) => (
-                      <Link
+                      <NavLink
                         key={item.name}
                         to={item.href}
                         className={classNames(
-                          item.current ? 'bg-gray-900 text-white' : 'text-gray-300 hover:bg-gray-700 hover:text-white',
```

Comment:
hey! What are you gaining from this? By removing **bg-gray-900**, it wouldn't be highlighted again.

Reply:
If it is not removed. The Home tab is  always highlighted with gray  by default. 
![Screenshot (28)](https://user-images.githubusercontent.com/120332757/226444729-8e3bd079-5843-4292-aecf-c86708e3c21d.png)